Tema2.js

```diff
@@ -0,0 +1,102 @@
+//problem 1
+console.log(problem1(3));
+function problem1(number) {
+    if (number % 2 != 0)
+        return number * number;
+    else
+        return number - 2;
+}
+
+
+//problem 2
+const savePrimes = primes => {
+    //console.log(savePrimes[1]);
+    const arrayPrimes = new Array;
+    let dimension = primes.length;
+    let contor = 0;
```

Comment:
nu este nevoie sa iti definesti un contor ca si in alte limbaje de programare, poti folosi metoda .push din Array:
if (div === primes[i]) {
            arrayPrimes.push(primes[i]);
        }

Reply:
(Ca si obs, contorul in array are locul lui, dar is cazuri foarte specifice de performanta)